notebooks/3_NeuralNetworks/autoencoder.ipynb

```diff
@@ -118,7 +122,23 @@
     "optimizer = tf.train.RMSPropOptimizer(learning_rate).minimize(cost)\n",
     "\n",
     "# Initializing the variables\n",
-    "init = tf.global_variables_initializer()"
+    "init = tf.initialize_all_variables()\n",
```

Comment:
This is regressing to deprecated code. Need to use `init = tf.global_variables_initializer()` instead